app/v2/controllers/DeleteHistoricUkPropertyAnnualSubmissionController.scala

```diff
@@ -102,4 +128,16 @@ class DeleteHistoricUkPropertyAnnualSubmissionController @Inject()(val authServi
       case NotFoundError => NotFound(Json.toJson(errorWrapper))
       case _             => unhandledError(errorWrapper)
     }
+
+  private def auditSubmission(
+      details: FlattenedGenericAuditDetail)(implicit endpointLogContext: EndpointLogContext, hc: HeaderCarrier, ec: ExecutionContext): Unit = {
+    val propertyType: String = endpointLogContext.endpointName match {
```

Comment:
Why do we need this logic? Add an extra parameter **_propertyType_** to **_auditSubmission_** function and pass it from the handleRequest function